src/main/java/net/sf/jabref/gui/preftabs/AppearancePrefsTab.java

```diff
@@ -182,7 +182,7 @@ public AppearancePrefsTab(JabRefPreferences prefs) {
         overrideFonts.addActionListener(e -> fontSize.setEnabled(overrideFonts.isSelected()));
 
         fontButton.addActionListener(
-                e -> new FontSelectorDialog(null, GUIGlobals.currentFont).getSelectedFont().ifPresent(x -> usedFont = x));
+                e -> new FontSelectorDialog(null, usedFont).getSelectedFont().ifPresent(x -> usedFont = x));
```

Comment:
The problematic part may be the `ifPresent`, as I am not sure if you can reassign a variable `usedFont` within a lambda `x -> usedFont = x` so that it `x` will be accessible from the outside.